### PR TITLE
Few wording/grammar suggestions/typo fixes

### LINKED ISSUE #23: typo
typo in the whitepaper in section 3. Eliminating third-party risk
says "elimnate" instead of eliminate second paragraph

### PR DESCRIPTION
**Pull request by Shadowfiend**
Broke these down into commits so they should be pretty easy to
tease apart/mutilate as needed.

Fixes #23.

### CODE REVIEW

File: keep.tex [hunk 1]
```diff
@@ -229,10 +229,10 @@ \subsection{Keep operations}
 \subsubsection{Creation and population}
 
 A contract, $Contract_{owner}$, requests a keep by publishing a
-request to the blockchain. Once a keep, $Keep_{accepted}$, accepted a
-request and is initialized off-chain, it will respond with a set of
-public keys the calling contract can use to communicate privately with
-the keep.
+request to the blockchain. Once a keep, $Keep_{accepted}$, has accepted a
+request and finished initializing off-chain, it will respond to the request
+with a set of public keys the calling contract can use to communicate privately
```
Comment (mhluongo) [inline keep.tex:234]: We've been keeping line length to 70

File: keep.tex [hunk 2]
```diff
@@ -293,7 +293,7 @@ \subsection{Secure multi party computation} \label{sMPC}
 the dealer, who can assemble the final output, without any participant
 learning more than their initial secret share.
 
-Intuitively, sMPC works like this.
+Intuitively, sMPC works like this:
```
Comment (mhluongo) [inline keep.tex:296]: :laughing: when I wrote this I expected it'd be edited out- even with the colon it's funny style for a more academic paper

File: keep.tex [hunk 3]
```diff
@@ -343,7 +343,7 @@ \subsection{sMPC and the blockchain}
 The advent of the blockchain enables secure usage of sMPC in
 adversarial scenarios. By using a public blockchain as an immutable
 ledger, sMPC can be made secure in the face of a dishonest
-supermajority \cite{spdz}, and with the requirement of a network
+supermajority \cite{spdz}, and, with the requirement of a network
```
Comment (mhluongo) [inline keep.tex]: +1 on the comma fix, but... would this be best split into two sentences?

**Review by mhluongo — APPROVED** [review]
:+1:

**Comment by mhluongo** [comment]
Thanks for this @Shadowfiend!

File: keep.tex (hunk 3 above)
Comment (Shadowfiend) [reply keep.tex]: I could see that, too. Can make that tweak if you'd like.

File: keep.tex (hunk 2 above)
Comment (Shadowfiend) [reply keep.tex:296]: <_<

File: keep.tex (hunk 1 above)
Comment (Shadowfiend) [reply keep.tex:234]: ! On it, sorry about that. My wrap is at 80, I kind of noticed but forgot to double-check.

File: keep.tex (hunk 3 above)
Comment (mhluongo) [reply keep.tex]: Let's wait, and do it in another PR if it's still an issue- we've got a bunch of changes in the pipeline re: system design

Comment (Shadowfiend) [reply keep.tex]: :100:

**Pull request merged**